### PR TITLE
HZ-0216:

### PR DESCRIPTION
**Pull request by CHesterHHH**
update SecurityConfig;
finished reset password;

### CODE REVIEW

File: src/main/java/com/eta/houzezbackend/controller/AgentController.java [hunk 1]
```diff
@@ -40,6 +40,12 @@ public Agent activeAgent(@RequestParam String token, @PathVariable Long id) {
         return agentService.setAgentToActive(token);
     }
 
+    @PatchMapping("/reset-password")
```
Comment (yuanningd) [inline src/main/java/com/eta/houzezbackend/controller/AgentController.java:43]: Verb in restful api?

Comment (CHesterHHH) [reply src/main/java/com/eta/houzezbackend/controller/AgentController.java:43]: This is hann's code. Jessey may fix this in the future.

File: src/main/java/com/eta/houzezbackend/controller/AgentController.java [hunk 2]
```diff
@@ -40,6 +40,12 @@ public Agent activeAgent(@RequestParam String token, @PathVariable Long id) {
...
+    public AgentGetDto resetPassword(@Valid @RequestBody AgentSignUpDto agentSignUpDto) {
```
Comment (yuanningd) [inline src/main/java/com/eta/houzezbackend/controller/AgentController.java:44]: Don’t you think the password can be reset by anyone?

Comment (CHesterHHH) [reply src/main/java/com/eta/houzezbackend/controller/AgentController.java:44]: Got it. Frontend pass me the token and new password.

**Review by Zachary1317 — APPROVED** [review]

**Pull request merged**